spec/lib/two_factor_authentication/models/two_factor_authenticatable_spec.rb

```diff
@@ -138,7 +138,7 @@ def instance.send_two_factor_authentication_code(code)
 
         it "returns uri with user's email" do
           expect(instance.provisioning_uri).
-            to match(%r{otpauth://totp/houdini@example.com\?secret=\w{32}})
+            to match(%r{otpauth://totp/houdini%40example.com\?secret=\w{32}})
```

Comment:
New ROTP versions output URL encoded URLs hence the test change.
This will be incompatible with old ROTP.